extensions/funqy/funqy-knative/deployment/src/test/java/io/quarkus/funqy/test/GreetingFunctions.java

```diff
@@ -9,12 +15,25 @@
     GreetingService service;
 
     @Funq
-    public Greeting greet(Identity name) {
-        String message = service.hello(name.getName());
-        Greeting greeting = new Greeting();
-        greeting.setMessage(message);
-        greeting.setName(name.getName());
-        return greeting;
+    public CompletionStage<Greeting> greet(Identity name) {
```

Comment:
Do not delete the old synchronous function.  Write a new test for CompletionStage<> return value.  Also, please duplicate the separate CompletionStage test in Lambda and HTTP.